pkgs/development/python-modules/kornia/default.nix

```diff
@@ -0,0 +1,43 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, packaging
+, pytestCheckHook
+, pytorch
+}:
+
+buildPythonPackage rec {
+  pname = "kornia";
+  version = "0.6.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-f/V8kxVRoaFGWqrB+mhCoqrWUPUaD5v2zwsPfW5ftZw=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "'pytest-runner'" ""
+  '';
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  propagatedBuildInputs = [
+    packaging
+    pytorch
+  ];
+
+  # pytest.ini is not published to pypi
+  doCheck = false;
```

Comment:
Could you please check if it's possible to execute the [tests](https://github.com/kornia/kornia/tree/master/test) when `fetchFromGitHub` is used to get the source?